app/src/main/java/com/capstone/weatherapp/CityListViewModel.kt

```diff
@@ -14,20 +14,12 @@ private const val UNITS = "imperial"
 class CityListViewModel(private val weatherRepo: WeatherRepository) : ViewModel() {
     val cityList = weatherRepo.cityListData
 
-    private val _eventNetworkError = MutableLiveData<Boolean>(false)
-    val eventNetworkError: LiveData<Boolean>
-        get() = _eventNetworkError
 
     fun getCityListFromRepo() {
         viewModelScope.launch {
             try {
```

Comment:
You no longer need the try catch here since it should be handled in the repository correctly